From a1b68db2783e407240e3e3ec0cda7c012da51c2b Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Mon, 29 Mar 2021 16:27:55 +0200
Subject: [PATCH 262/529] dt-bindings: backlight: add new property
 default-brightness-level

Add new property to set a brightness by default at probe.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: I813e18f740445a407cd333bd4f1754006b97b6c2
---
 .../bindings/leds/backlight/gpio-backlight.yaml          | 9 +++++++++
 1 file changed, 9 insertions(+)

--- a/Documentation/devicetree/bindings/leds/backlight/gpio-backlight.yaml
+++ b/Documentation/devicetree/bindings/leds/backlight/gpio-backlight.yaml
@@ -23,6 +23,15 @@ properties:
     description: enable the backlight at boot.
     type: boolean
 
+  default-brightness-level:
+    description:
+      The default brightness level (index into the array defined by the
+      "brightness-levels" property).
+    $ref: /schemas/types.yaml#/definitions/uint32
+
+dependencies:
+  default-brightness-level: [brightness-levels]
+
 required:
   - compatible
   - gpios
